File: 백준/Bronze/1924. 2007년/2007년.java
import java.io.EOFException;
import java.util.*;

public class Main {
    public static void main(String[] args) {

        Scanner sc=new Scanner(System.in);

        int x=sc.nextInt();
        int y=sc.nextInt();

        //String[] weeks={'SUN','MON','TUE','WED','TUE','FRI','SAT'};

        int m[]={31,28,31,30,31,30,31,31,30,31,30,31};

        String[] weeks={"SUN","MON","TUE","WED","THU","FRI","SAT"};

        if(x==1){
            System.out.println(weeks[y%7]);
        }
        else{
            int day_index=y;
            for(int i=0;i<x-1;i++){
                day_index+=m[i];
            }
            System.out.println(weeks[day_index%7]);
        }

    }
}

File: 백준/Bronze/1924. 2007년/2007년.py
x,y=map(int,input().split())
m={
            1:31,
            2:28,
            3:31,
            4:30,
            5:31,
            6:30,
            7:31,
            8:31,
            9:30,
            10:31,
            11:30,
            12:31
            }

days=['SUN','MON','TUE','WED','THU','FRI','SAT']




if(x==1):
    print(days[y%7])
else:
    day_index=y
    for i in range(1,x):
        day_index+=m[i]
    print(days[day_index%7])
